### ISSUE TITLE
Feature request: Show result in Rendered view

### ISSUE THREAD

**Issue by SilvM** [CalebJohn/joplin-math-mode#17]
Thank you for this plugin. It was the feature I didn't know I wanted until I saw it in the list of plugins.

I usually use Joplin as a notebook via an external editor, but also as a presentation tool both which rely on the Rendered view. In these cases it would be nice if the results would also be visible in the rendered markup.

**Comment by CalebJohn** [comment]
Agreed! Which is why it's on the [Todo list](https://github.com/CalebJohn/joplin-math-mode#todo). 

If you need a quick fix, you can take a look at [this PR](https://github.com/CalebJohn/joplin-math-mode/pull/14) that implements rendering. Unfortunately, it doesn't support configuration. I spent a day working on it, but have not had enough time to actually complete anything. I still plan on getting there eventually, but it might be another few months.

If you happen to want to take on the challenge, I'd be happy to help in any way I can.

**Comment by SilvM** [comment]
I read the todo list before but I didn't connect the dots it referred to the same thing 😁 .

I dabbed in web development here and there but I'm no JS developer tho, not sure where I could start contributing to that PR. By support configuration you mean to have a toggle for the markup rendering in settings?

**Comment by CalebJohn** [comment]
That might be a configuration option that I'll add. But I just mean in general any of the options in the settings menu. Precision, and format are examples user configuration. Basically, the rendered results need to match the preview results, no matter what a users settings are.

**Comment by banan314** [comment]
I read the todo about latex and rendering in view. Maybe would it be possible to render in the exported PDF or HTML as well? :pleading_face:

**Comment by CalebJohn** [comment]
Math is now rendering in the viewer as of v1.0.0!
